modules/util/source-formatter/src/main/resources/sourcechecks.xml

```diff
@@ -1666,6 +1666,10 @@
 			<description name="Replace getLayoutDisplayPageProvider by getLayoutDisplayPageProviderByClassName" />
 			<weight>1</weight>
 		</check>
+		<check name="UpgradeJavaLayoutServicesCheck">
+			<category name="Upgrade" />
+			<description name="Fill the new parameters of the method `addLayout` and `updateLayout` of `LayoutLocalService` and `LayoutLocalServiceUtil` classes" />
+		</check>
```

Comment:
@TamyrisTorres, do you have sure that the pattern happens only in these two classes? In the `LayoutService` and `LayoutServiceUtil` doesn't happen?